confgenerator/fluentbit/processors.go

```diff
@@ -110,6 +111,9 @@ func ParserComponentBase(TimeFormat string, TimeKey string, Types map[string]str
 	if TimeKey != "" {
 		parser.Config["Time_Key"] = TimeKey
 	}
+	if TimeOffset != "" {
```

Comment:
Actually, the omit-if-zero idea might be more appropriate here instead:

```suggestion
	if TimeOffset != "" && TimeOffset != "+0000" && TimeOffset != "-0000" {
```